src/components/Spacing/Spacing.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = () => css`
+  label: spacing;
+`;
+
+const marginBottomStyles = ({ theme, bottom }) =>
+  bottom &&
+  css`
+    label: spacing--margin-bottom;
+    margin-bottom: ${theme.spacings.giga};
```

Comment:
Shouldn't these be responsive? Not sure if there is any guideline on this from design. But maybe this would be a time to ask?

I'm also wondering if you should be able to specify values for amount of spacing. After all, spacings are different in different situations. The grid uses a certain gutter. Inputs have specific bottom margin. Buttons have horizontal margin. Etc.

But maybe you didn't do this just yet, because we are going to discuss it on Friday?